pkg/synopsysctl/cmd_migrate.go

```diff
@@ -298,6 +299,29 @@ func migrateAlert(namespace string) error {
 		alertNamespace := alert.Spec.Namespace
 		log.Infof("migrating Alert '%s' in namespace '%s'...", alertName, alertNamespace)
 
+		// default images
+		defaultAlertImages := []string{
+			"docker.io/blackducksoftware/blackduck-alert:3.1.0",
+			"docker.io/blackducksoftware/blackduck-alert:4.0.0",
+			"docker.io/blackducksoftware/blackduck-alert:4.1.0",
+			"docker.io/blackducksoftware/blackduck-alert:4.2.0",
+		}
+		defaultCfsslImage := "docker.io/blackducksoftware/blackduck-cfssl:1.0.0"
+
+		// get the current value of the images using reflect
+		alertImage := reflect.ValueOf(alert.Spec).Elem().FieldByName("AlertImage").Interface().(string)
```

Comment:
Need to verify this is ok.  Can check with .IsValid or should be able to use .String() instead of .Interaface().(string) and it should return empty string if not valid.

Reply:
fixed